robocup_ws/src/Planner/src/player_controller.py

```diff
@@ -465,17 +456,19 @@ def distance_judge(self, game_info: list, init_distance = None) -> [bool, int, i
 
         return False, -1, -1
 
-    def avoid_obstacle(self, game_info: list, team_id, obstacle_player_id) -> PlayerCommand:
+    @staticmethod
+    def avoid_obstacle(game_info: list, team_id, obstacle_player_id) -> PlayerCommand:
         team = game_info[0]
         obstacle_team = game_info[team_id]
         player_id = game_info[2]
         my_pos = team.players_positions_efcs[player_id]
         safe_pos = obstacle_team.players_positions_wcs[obstacle_player_id]
 
-        lv, rv = self.go_around_the_point(my_pos, safe_pos, 0.3)
+        lv, rv = PlayerController.go_around_the_point(my_pos, safe_pos, 0.3)
         return PlayerCommand(lv, rv, 0)
 
-    def go_around_the_point(self, robot_state: Position, go_around_point: Position, radius, direction=1, R_vel=3):
+    @staticmethod
+    def go_around_the_point(robot_state: Position, go_around_point: Position, radius, direction=1, R_vel=3):
```

Comment:
why you can't reuse the function from the ```go_to_actions.py```?
https://github.com/szgula/UofG_Robotics_TDP/blob/019a4f8273a2d70fcc9b73dfc27cfdca2cb64707/robocup_ws/src/Planner/src/BasicCommonActions/go_to_action.py#L139

Reply:
Because in this part, I was going to improve this model and add something : ). So I put here.

Reply:
why not improve it in the ```go_to_actions.py``` and have one better/more robust code? 2 times less maintenance effort and immediate improvement for all players ;)

The golden rule in software development is to write code that can be reused by other people/projects/modules - start now and you would have a nice story for future interviews ;)